backend/tests/api/test_profile.py

```
@@ -0,0 +1,193 @@
+        second = await client.post("/api/v1/profile", json=payload)
+
+        assert first.status_code == 201
+        assert second.status_code == 409
+        assert "already exists" in second.json()["detail"].lower()
+
+    @pytest.mark.asyncio
+    async def test_create_profile_conflict_error_maps_to_409(
+        self, client: AsyncClient
+    ) -> None:
+        class BrokenCreateProfileService:
+            async def create_profile(self, payload: dict[str, Any]) -> dict[str, Any]:
```

Ruff (0.15.1)

[warning] Unused method argument: `payload`

(ARG002)